src/main/java/com/elmakers/mine/bukkit/action/builtin/SimulateAction.java

```diff
@@ -3,789 +3,17 @@
 import com.elmakers.mine.bukkit.action.BaseSpellAction;
 import com.elmakers.mine.bukkit.api.action.CastContext;
 import com.elmakers.mine.bukkit.api.spell.SpellResult;
-import com.elmakers.mine.bukkit.block.MaterialAndData;
-import org.bukkit.Location;
-import org.bukkit.Material;
-import org.bukkit.block.Block;
-import org.bukkit.block.BlockFace;
-import java.util.ArrayList;
-import java.util.List;
 
 public class SimulateAction extends BaseSpellAction {
```

Comment:
Same question/comment as AnimateAction - these were meant to replace AnimateSpell and SimulateSpell, but I never went anywhere with them.